content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/update_status.yaml

```diff
@@ -17,7 +20,7 @@ object:
       datatype: string
       priority: 1
       owner: 
-      default_value: 
+      default_value: smartvm
```

Comment:
@billfitzgerald0120 
Should this be smartvm?

Reply:
@mkanoor Not sure how this got in there but I deleted it.